srh_jamaica.py

```diff
@@ -0,0 +1,70 @@
+import os
+import subprocess
+import sys
+from rapidpro_flow_tools import flow_converter
+
+def main(credentials = None, token = None):
+
+    source_files = ["srh_registration", "srh_entry", "srh_content", "srh_safeguarding"]
+    spreadsheet_IDS = ["1yett-Rfzb9Ou8IQ1kwtrKPN_auhM-lk66r9gkqNV1As", "19xvYfwWKA1hT5filGPWYEobQL1ZFfcFbTj1-aJCN8OQ", "1hOlgdqjmXZgl51L1olt357Gfiw2zRHNEl98aYTf8Hwo", "1A_p3cb3KNgX8XxD9MlCIoa294Y4Pb9obUKfwIvERALY"]
```

Comment:
These two lists should be combined, so that it's clear what the relationship between them is.

```python
sources = [
    {"filename": "srh_registration", "spreadsheet_id": "1yett-Rfzb9Ou8IQ1kwtrKPN_auhM-lk66r9gkqNV1As"},
    // etc...
]

// usage
for source in sources:
    source_file_name = source["filename"]
    spreadsheet_id  = source["spreadsheet_id"]
```

Alternatively, a class could be created.

```python
class Source:
    def __init__(self, filename, spreadsheet_id):
        self.filename = filename
        self.spreadsheet_id = spreadsheet_id

sources = [
    Source("srh_registration", "1yett-Rfzb9Ou8IQ1kwtrKPN_auhM-lk66r9gkqNV1A"),
    // etc...
]

for source in sources:
    source_file_name = source.filename
    sid = source.spreadsheet_id
```

Reply:
have combined as suggested